cmd/apiserver/app/server/run_server.go

```diff
@@ -149,9 +151,29 @@ func runEtcdServer(opts *ServiceCatalogServerOptions) error {
 		return fmt.Errorf("error completing API server configuration: %v", err)
 	}
 
+	// Install healthz checks before calling PrepareRun.
+	// This is an example. PingHealtz is installed by the default config.
+	// server.GenericAPIServer.AddHealthzChecks(healthz.PingHealthz)
```

Comment:
Yea, commented out, until I figure out how to make a real one. PingHealthz is added by default config of the genericapiserver. If it gets added again it's an error in the core golang libraries for a mux. Now that I think about it, possible upstream feature to add that checks if the same name exists already before adding it again.